packages/cape/src/encrypt.ts

```diff
@@ -82,8 +79,7 @@ export async function forgeRsaEncrypt(plainText: Uint8Array, key: string): Promi
  * @param plainText The plain text input to encrypt.
  */
 export async function capeEncrypt(capeKey: string, plainText: string): Promise<string> {
```

Comment:
We make the `plainText` argument accept the same type as the `aesEncrypt` `plainText` argument